base/docs/helpdb/Dates.jl

```diff
@@ -49,11 +50,55 @@ All characters not listed above are treated as delimiters between date and time
+
+Construct a string by using a `TimeType` and applying the pattern provided by the
+`format` string. The following codes can be used for constructing the `format` string:
+
+| Code       | Example   | Comment                                                      |
+|:-----------|:----------|:-------------------------------------------------------------|
+| `y`        | 6         | Numeric year with a fixed width                              |
+| `m`        | 1, 12     | Numeric month with a minimum width                           |
+| `u`        | Jan       | Month name shortened to 3-chars according to the `locale`    |
+| `U`        | January   | Full month name according to the `locale` keyword            |
+| `d`        | 1, 31     | Day of the month with a minimum width                        |
+| `H`        | 0, 23     | Hour (24-hour clock) with a minimum width                    |
+| `M`        | 0, 59     | Minute with a minimum width                                  |
+| `S`        | 0, 59     | Second with a minimum width                                  |
+| `s`        | 000, 500  | Millisecond with a minimum width of 3                        |
+| `e`        | Mon, Tue  | Abbreviated days of the week                                 |
+| `E`        | Monday    | Full day of week name                                        |
+
+The number of sequential code characters indictate the width of the code. A format of
+`yyyy-mm` specifies that the code `y` should have a width of four while `m` a width of two.
+Codes yielding a numeric string will have associated width mode: fixed or minimum. Fixed
+mode will left-pad the value with zeros when it is shorter than the width and truncate the
+value when longer. Minimum mode works the same as fixed except that it does not truncate
+values longer than the width.
+
+Any characters not listed above are left in the output string where they are. To create a
+string of "1996-01-15T00:00:00" would have a `format` string like "yyyy-mm-ddTHH:MM:SS".
+"""
+Dates.format(dt::Dates.TimeType, format::AbstractString)
+
+
+"""
+    DateFormat(format::AbstractString) -> DateFormat
+
+Construct a date formatting object that can be used for parsing or formatted date
+strings. For details on the formatting syntax see details for see
+[`DateTime(::AbstractString, ::AbstractString)`](:ref:`parsing <man-date-parsing>`) and
+[`format`](:ref:`formatting <man-date-formatting>`).
+"""
+Dates.DateFormat(f::AbstractString,locale::AbstractString="english")
+
+"""
     DateTime(dt::AbstractString, df::DateFormat) -> DateTime
 
-Similar form as above for parsing a `DateTime`, but passes a `DateFormat` object instead of
-a raw formatting string. It is more efficient if similarly formatted date strings will be
-parsed repeatedly to first create a `DateFormat` object then use this method for parsing.
+Similar form as above for parsing a `DateTime`, but uses a
```

Comment:
I know I also found it hard to phrase this in my own code. Maybe just "similar as above" (or "similar to"? else "same as" would work). I would even remove the redundant "for parsing a DateTime".